### PR TITLE
CN-5142: removed ZEC and BTG

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/utils/crypto-util.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ function isERC20(currency) {
       "tusd",
       "gusd",
       "usdc",
-      "orbs",
```
Comment (fjovakaric) [inline app/utils/crypto-util.js:23]: check if we are using this anywhere

File: app/utils/crypto-util.js [hunk 2]
```diff
@@ -230,18 +222,6 @@ export function getBlockExplorerLink(transaction) {
...
-      return {
-        link: BLOCKEXPLORERS.zec && `${BLOCKEXPLORERS.zec}${tId}`,
-        text: "chain.so",
-      };
-    // BTG
-    case "btg":
-      return {
-        link: BLOCKEXPLORERS.btg && `${BLOCKEXPLORERS.btg}${tId}`,
-        text: "btgexplorer",
-      };
```
Comment (fjovakaric) [inline app/utils/crypto-util.js:244]: don't remove

File: app/constants/DATA.js [hunk 3]
```diff
@@ -16,13 +16,11 @@ const KYC_STATUSES = {
 
 const BLOCKEXPLORERS_STAGING = {
   ltc: "https://chain.so/tx/LTCTEST/",
-  zec: "https://chain.so/tx/ZECTEST/",
```
Comment (fjovakaric) [inline app/constants/DATA.js:19]: don't remove

File: app/utils/crypto-util.js (hunk 1 above)
Comment (srkela) [reply app/utils/crypto-util.js:23]: It's used in `WithdrawCreateAddress` for `InfoBox`
